app.rb

```diff
@@ -0,0 +1,156 @@
+require './student'
+require './teacher'
+require './book'
+require './rental'
+require './person'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+    @exit = false
+  end
+
+  def run
+    user_input until @exit
+  end
+
+  def display_choice
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
+    puts '4 - Create a book'
+    puts '5 - Create a rental'
+    puts '6 - List all rentals for a given id'
+    puts '7 - Exit'
+  end
+
+  # rubocop:disable Metrics/CyclomaticComplexity
+  def user_input
+    puts '=============================='
+    display_choice
+    puts '=============================='
+
+    puts 'Enter your choice'
```

Comment:
- [ ] [OPTIONAL] For a better UI, kindly add a column after the words you are printing for user`:`
For example here you are printing `Enter your choice`.
It would be better to make it `Enter your choice:`
Kindly implement this on other prints as well.
###  Take a look at your implementation
![Screenshot from 2022-10-15 14-53-48](https://user-images.githubusercontent.com/58692590/195990467-835929fc-4b28-474d-aeff-3071ed1c6a56.png)